src/app/views/Home/Components/Pod.elm

```diff
@@ -0,0 +1,68 @@
+module Home.Components.Pod exposing (..)
```

Comment:
Does `exposing (..)` mean it's exposing all the stuff from `Home.Components.Pod`?